frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/RelationshipEdge/RelationshipEdge.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  BaseEdge,
+  type Edge,
+  type EdgeProps,
+  getSmoothStepPath,
+} from '@xyflow/react'
+
+import type { FC } from 'react'
+
+export type RelationshipEdgeType = Edge
+
+type Props = EdgeProps<RelationshipEdgeType>
```

Comment:
`RelationshipEdgeType` is not used other file, so I thought it looked good to delete.

```suggestion
type Props = EdgeProps<Edge>
```

Reply:
Oh, thank you so much🙏.